api/src/pcapi/core/fraud/dms/api.py

```diff
@@ -38,12 +38,19 @@ def start_dms_fraud_check(
             type=fraud_models.FraudCheckType.DMS,
             thirdPartyId=application_id,
             resultContent=source_data.dict(),
-            status=fraud_models.FraudCheckStatus.PENDING,
+            status=fraud_models.FraudCheckStatus.STARTED,
             eligibilityType=source_data.get_eligibility_type(),
         )
         pcapi_repository.repository.save(fraud_check)
 
-    if fraud_check.status != fraud_models.FraudCheckStatus.PENDING:
+    if fraud_check.status != fraud_models.FraudCheckStatus.STARTED:
```

Comment:
`if fraud_check.status not in (fraud_models.FraudCheckStatus.STARTED, PENDING)`

+ créer un ticket pour enlever "PENDING" une fois que le code est en place et que les données sont rattrapées.